src/clients/api/queries/getPendingXvs.ts

```diff
@@ -0,0 +1,19 @@
+import BigNumber from 'bignumber.js';
+import { VaiVault } from 'types/contracts';
+
+export interface IGetPendingXvsInput {
+  vaiVaultContract: VaiVault;
+  accountAddress: string;
+}
+
+export type GetPendingXvsOutput = BigNumber;
+
+const getPendingXvs = async ({
```

Comment:
This is more of a question than a suggestion: seeing we've now adopted the pattern of suffixing variables with their unit, shall we do the same with queries?